src/Lykke.AlgoStore.Service.Security.Core/Utils/FuncExtensions.cs

```diff
@@ -7,7 +7,7 @@ namespace Lykke.AlgoStore.Service.Security.Core.Utils
 {
     public static class FuncExtensions
     {
-        public static async Task LogElapsedTime(this Func<Task> action, ILog log, string clientId)
+        public static async Task LogElapsedTime(this ILog log, string clientId, Func<Task> action)
```

Comment:
This should be renamed to LogElapsedTimeAsync to better follow convention around the project